src/store/pluginStore.ts

```diff
@@ -0,0 +1,27 @@
+import { Plugin } from '@typings/plugin';
+import { StoreCreator } from '.';
+import { getPlugin } from '@provider/plugins';
+
+export type PluginStore = {
+  selectedPlugin?: Plugin.PluginItem;
```

Comment:
```suggestion
  pluginItem?: Plugin.PluginItem;
```

Reply:
You have to change it in other files that use this car as well.